BankSDK/GiniBankSDKExample/GiniBankSDKExample.xcodeproj/project.pbxproj

```diff
@@ -998,6 +1103,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = net.gini.banksdk.example;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE_SPECIFIER = "";
+				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "Gini Bank SDK Example App Ad-Hoc Distribution";
```

Comment:
[clarification]: we are going to set up manually all the profiles?

Reply:
I set those only for release mode and 2 targets.
It'll not affect our debug installations.